content/c-sharp/concepts/math-functions/terms/cos/cos.md

```diff
@@ -22,15 +22,17 @@ The **`Math.Cos()`** class method returns the cosine of a given angle.
 Math.Cos(angle);
 ```
 
-The `Math.Cos()` method takes only one `double` type parameter - `angle` - an angle in radians. The `Math.Cos()` method returns the cosine of the `angle` as a `double` value or `NaN` (not a number) if the `angle` equals to:
+The `Math.Cos()` method takes only one parameter, `angle`, an angle in radians of type `double`. The method returns the cosine of the `angle` as a `double` value or `NaN` (not a number) if the value of `angle` equals:
 
-- `NaN`,
-- `NegativeInfinity`, or
-- `PositiveInfinity`.
+- `NaN`
+- `NegativeInfinity`
+- `PositiveInfinity`
+
+> **Note:** Depending on the used operating system or architecture, the exact result or the input range of the `Math.Cos()` method may differ due to the differences in the C runtime environment.
```

Comment:
I think we should move the note below the example text (after line 35). Also with the note we can forego the rounding step.